appveyor.yml

```diff
@@ -12,10 +12,35 @@ environment:
      PYTHON_VERSION: "3.6.x"
      PYTHON_ARCH: "64"
 
+   - PYTHON: "C:\\Python37"
+     PYTHON_VERSION: "3.7.x"
+     PYTHON_ARCH: "32"
+
+   - PYTHON: "C:\\Python37-x64"
+     PYTHON_VERSION: "3.7.x"
+     PYTHON_ARCH: "64"
+
+   - PYTHON: "C:\\Python38"
+     PYTHON_VERSION: "3.8.x"
+     PYTHON_ARCH: "32"
+
+   - PYTHON: "C:\\Python38-x64"
+     PYTHON_VERSION: "3.8.x"
+     PYTHON_ARCH: "64"
+
 init:
   - "ECHO %PYTHON% %PYTHON_VERSION% %PYTHON_ARCH%"
 
 install:
+  - ps: |
+      Set-PSDebug -Trace 1
+      if ($Env:PYTHON_VERSION.StartsWith("3.8")) {
```

Comment:
Appveyor has 3.8 as of [a few days ago](https://github.com/appveyor/ci/issues/3142#issuecomment-552049896), I think, as long as we use an image >= vs 2015

Reply:
removed, worked the first time!